### PR TITLE
[GEN-763] Simplify Auth

### PR DESCRIPTION
**Pull request by redpepper-nag**
**Overview**
- auth is no longer a navigator between signin/signup
- visual changes to make input form simpler to navigate
- google auth shows on both platforms but has null handler for ios
- apple auth shows on ios but has empty handler
- commented out the toggles for setter and nyuComp

**Visuals**
IOS:
IMG_2402

Android:
420190625_761933765988793_1575006839278061959_n

### CODE REVIEW

File: trk/src/App.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ function App(props) {
 
   return (
     <PaperProvider>
-      {currentUser ? <AppNavigator /> : <AuthNavigator />}
+      {currentUser ? <AppNavigator /> : <AuthScreen />}
       <Toast/>
```
Comment (redpepper-nag) [inline trk/src/App.js:30]: AuthNavigator renamed to AuthScreen since it is no longer a nav

File: trk/src/Components/SignInUp.js [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+    auth().sendPasswordResetEmail(email)
+      .then(() => {
+        Alert.alert("Password Reset", "Check your email to reset your password.");
+      })
+      .catch(error => {
+        Alert.alert("Reset Password Error", error.message);
+      });
+  };
+
+  const onGoogleButtonPressAndroidOnly = Platform.OS === 'android' ? onGoogleButtonPress : undefined;
```
Comment (redpepper-nag) [inline trk/src/Components/SignInUp.js:115]: on IOS the onPress handler for google auth is null (will be fixed later)

File: trk/src/Components/SignInUp.js [hunk 3]
```diff
@@ -0,0 +1,292 @@
+import React, { useState } from 'react';
```
Comment (redpepper-nag) [inline trk/src/Components/SignInUp.js:1]: This entire file is just a merge of SignIn and SignUp (which were delete above)

File: trk/src/Components/SignInUp.js [hunk 4]
```diff
@@ -0,0 +1,292 @@
...
+        </TouchableOpacity>
+        {Platform.OS === 'ios' && (
+    <TouchableOpacity
+      onPress={onAppleButtonPress} // Ensure you have implemented onAppleButtonPress for handling the sign-in process
+      style={styles.alternateSignInButton}
+    >
+      <Image source={require('../../assets/apple.png')} style={styles.appleIcon} />
+      <Text style={styles.alternateSignInButtonText}>Continue with Apple</Text>
+    </TouchableOpacity>
+  )}
```
Comment (redpepper-nag) [inline trk/src/Components/SignInUp.js:171]: apple button only shows for ios platform

File: trk/src/Navigation/AuthScreen.js [hunk 5]
```diff
@@ -1,23 +1,23 @@
 import React from 'react';
 import { createStackNavigator } from '@react-navigation/stack';
 import { NavigationContainer } from '@react-navigation/native';
-import SignInUpToggle from '../Components/SignInUpToggle';
+import SignInUp from '../Components/SignInUp';
```
Comment (redpepper-nag) [inline trk/src/Navigation/AuthScreen.js:4]: remamed SignInUpToggle to SignInUp

**Review by abhipi — APPROVED** [review]
Looks great!

**Pull request merged**